app/models/middleware_messaging.rb

```diff
@@ -0,0 +1,6 @@
+class MiddlewareMessaging < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server, :foreign_key => "server_id"
```

Comment:
we need to reflect hawkular inventory, so if this is reported as duplicated, we should show as duplicated imo. @pilhuhn what shall you say?